src/main/resources/assets/paths/mailByRange.yaml

```diff
@@ -0,0 +1,46 @@
+get:
+  summary: Get Mail Messages by Date Range.
+  description: |
+    Admin only feature returns MailMessage list filtered by Date Range.
+  parameters:
+    - name: start
+      in: query
+      description: Earliest Date to search in MM/dd/yyyy format. (e.g. 05/19/2019)
+      required: true
+      schema:
+        type: string
+    - name: end
+      in: query
+      description: Latest Date to search in MM/dd/yyyy format. (e.g. 05/20/2022)
+      required: true
+      schema:
+        type: string
```

Comment:
what if it's optional, and defaults to the time of being called if not specified? similarly for start, could be optional and default to a time far in the past.